docs/content-services/components/search-date-range-advanced.component.md

```diff
@@ -0,0 +1,58 @@
+
+Represents an advanced date range component for
+the [SearchAdvancedDateRangeTabbedComponent](search-date-range-advanced-tabbed.component.md).
+
+![Date Range Advanced Widget](../../docassets/images/search-date-range-advanced.png)
+
+## Basic usage
+
+```html
+
+<adf-search-date-range-advanced></adf-search-date-range-advanced>
+```
+
+## Class Members
+
+### Properties
+
+| Name         | Type                    | Description                                                                                                                                                                                                                                                |
+|--------------|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| field        | string                  | Field to apply the query to. Required value                                                                                                                                                                                                                |
+| maxDate      | string                  | A fixed date (default format: dd-MMM-yy) or the string `"today"` that will set the maximum searchable date. Default is today.                                                                                                                              |
+| dateFormat   | string                  | Date format. Dates used by the datepicker are Javascript Date objects, using [date-fns](https://date-fns.org/v2.30.0/docs/format) for formatting, so you can use any date format supported by the library. Default is 'dd-MMM-yy (sample date - 07-Jun-23) |
+| initialValue | SearchDateRangeAdvanced | Initial value for the component                                                                                                                                                                                                                            |
+
+### Events
+
+| Name                | Type                                                                                           | Description                                                                                                                                    |
+|---------------------|------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| updatedQuery        | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<string>`                           | Emitted whenever a change is made in the component values. Emits the updated query based on the change made.                                   |
```

Comment:
that event seems not exist anymore in component

Reply:
These have been removed in [this](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/commits/bdaa0e52fd3a31ba7c3802c6cdc6a27a733d6057) commit